src/ceph-volume/ceph_volume/process.py

```diff
@@ -114,7 +114,8 @@ def run(command, **kw):
     )
 
     if stdin:
-        process.communicate(stdin)
+        process.stdin.write(stdin)
+        process.stdin.close()
```

Comment:
It is recommended not to write directly like this to stdin because: 

> Use communicate() rather than .stdin.write, .stdout.read or .stderr.read to avoid deadlocks due to any of the > other OS pipe buffers filling up and blocking the child process.

From: https://docs.python.org/2/library/subprocess.html#subprocess.Popen.communicate

IMO, this should be removed, we can't do this nicely and keep the terminal output flowing. This is currently OK as nothing else is calling `process.run` and using stdin here. There is another place where `process.call` is being used when trying stdin.

The right thing would be to just remove the `stdin` ability from here.